### DISCUSSION TITLE
Potential UX paper-cuts when working w/ object overlays

### DISCUSSION THREAD

**Discussion by vaidaslamanauskas** [tinacms/tinacms#3784] [Ideas]
I caught this issue myself even after working a bit w/ Tina and then my editor immediately got it too.

When adding a new object, overlay opens and the pull is immediately towards large blue **Save** action in hope to save that single object, however it of course saves the entire new document. Since there's no publishing flow, for those who have auto-deployments set up, not so good.

I believe there's a potential for quick-wins here:

**01** Overlay object form on top of the action bar and provide bigger area for closing.

![yuba world_admin_](https://user-images.githubusercontent.com/4195325/230112738-b747bd20-8e4f-4b4a-a347-576590509ea2.png)

**02** Use _back pattern_ together w/ the object item titles.

![yuba world_admin_ (1)](https://user-images.githubusercontent.com/4195325/230113913-b20fa270-086c-4d8e-99dd-1487fdb957ff.png)

**03** Not so _quick_, but open it w/ context of the main  editing area visible, could even be similar pattern to Media Manager

![Frame 1](https://user-images.githubusercontent.com/4195325/230116907-b4c3870e-3a17-405a-a326-cbf84dddb1ff.jpg)

Let me know what you think, I'm sure there's a reasoning and better possible solutions 🙇

**Comment by spbyrne** [comment]
Great feedback, I think this is a valid concern. I think we'll have to give the solution some consideration, but all three options you're presenting here seem reasonable. I think maybe some sort of combination of 1 & 2 might work.

**Comment by vaidaslamanauskas** [comment]
Good to hear! Lmk if I can be of any assistance.
